README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
```

Comment:
Oh, I remember: I renamed it in the end to be consistent with the naming of the previous sub-chapter.